modules/contact/doc/config.yml

```diff
@@ -28,6 +29,9 @@ Extensions:
         shortcuts: !include framework/doc/globals.yml
     MooseDocs.extensions.acronym:
         acronyms: !include framework/doc/acronyms.yml
+    MooseDocs.extensions.bibtex:
+        duplicates:
+            - puso2008segment # also in framework
```

Comment:
is this preferred over removing it from the contact bib file?

Reply:
imo yes it's more future proof

Reply:
I like having docs build separately and cleanly, as the modules themselves do. We've done his historically, so I am attempting to keep it up when I notice it.